homeassistant/components/vesync/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "VeSync",
   "documentation": "https://www.home-assistant.io/integrations/vesync",
   "codeowners": ["@markperdue", "@webdjoe", "@thegardenmonkey"],
-  "requirements": ["pyvesync==2.0.3"],
+  "requirements": ["pyvesync==2.1.1"],
```

Comment:
The source version hasn't been updated.

https://github.com/webdjoe/pyvesync/blob/8861ec5fe27db9b3734744f6c31602fb10e42c55/setup.py#L13

Please ask the maintainer to update the version and tag a github release for each release that is published on PyPI. This allows the community to track changes.

Reply:
reached out to the maintainer, I'll comment back here when it's done

Reply:
Looks like github is out of sync with pypi: https://pypi.org/project/pyvesync/2.1.1/

 I have opened an issue to automate this on the maintainers side.